docs/contributing.rst

```diff
@@ -70,50 +70,50 @@ Thank you for your help!
 Running the compiler tests
 ==========================
 
-There is a script at ``scripts/tests.sh`` which executes most of the tests and
+The ``scripts/tests.sh`` script executes most Solidity tests and
 runs ``aleth`` automatically if it is in the path, but does not download it,
-so it most likely will not work right away. Please read on for the details.
+so you need to install it first. Please read on for the details.
 
-Solidity includes different types of tests. Most of them are bundled in the application
-called ``soltest``. Some of them require the ``aleth`` client in testing mode,
-some others require ``libz3`` to be installed.
+Solidity includes different types of tests, most of them bundled into the ``soltest``
+application. Some of them require the ``aleth`` client in testing mode, others require ``libz3``.
 
-To run a basic set of tests that neither require ``aleth`` nor ``libz3``, run
-``./scripts/soltest.sh --no-ipc --no-smt``. This script will run ``build/test/soltest``
+To run a basic set of tests that require neither ``aleth`` nor ``libz3``, run
+``./scripts/soltest.sh --no-ipc --no-smt``. This script runs ``build/test/soltest``
 internally.
 
-The option ``--no-smt`` disables the tests that require ``libz3`` and
+The option ``--no-smt`` disables the tests that require ``libz3``, and
 ``--no-ipc`` disables those that require ``aleth``.
 
-If you want to run the ipc tests (those test the semantics of the generated code),
+If you want to run the ipc tests (that test the semantics of the generated code),
 you need to install `aleth <https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/aleth_2018-06-20_artful>`_ and run it in testing mode: ``aleth --test -d /tmp/testeth`` (make sure to rename it).
 
-Then you run the actual tests: ``./scripts/soltest.sh --ipcpath /tmp/testeth/geth.ipc``.
+To run the actual tests, use: ``./scripts/soltest.sh --ipcpath /tmp/testeth/geth.ipc``.
 
-To run a subset of tests, filters can be used:
+To run a subset of tests, you can use filters:
 ``./scripts/soltest.sh -t TestSuite/TestName --ipcpath /tmp/testeth/geth.ipc``,
 where ``TestName`` can be a wildcard ``*``.
 
 The script ``scripts/tests.sh`` also runs commandline tests and compilation tests
 in addition to those found in ``soltest``.
 
-The CI even runs some additional tests (including ``solc-js`` and testing third party Solidity frameworks) that require compiling the Emscripten target.
+Travis CI runs some additional tests (including ``solc-js`` and testing third party Solidity frameworks) that require compiling the Emscripten target.
```

Comment:
Not only travis. I think we should not mention which CI runs what here.

Reply:
@chriseth Sorry, that was just a bad merge.